meta-ibm/meta-witherspoon/recipes-phosphor/nvme/phosphor-nvme.bb

```diff
@@ -0,0 +1,26 @@
+PR = "r1"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${PHOSPHORBASE}/COPYING.apache-2.0;md5=34400b68072d710fecd0a2940a0d1658"
+
+inherit meson pkgconfig
+inherit obmc-phosphor-dbus-service
+inherit obmc-phosphor-systemd
+inherit phosphor-dbus-yaml
+inherit pythonnative
+
+DEPENDS += "sdbusplus"
+DEPENDS += "phosphor-dbus-interfaces"
+DEPENDS += "sdeventplus"
+DEPENDS += "phosphor-logging"
+DEPENDS += "sdbusplus-native"
+DEPENDS += "autoconf-archive-native"
+DEPENDS += "nlohmann-json"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+SRC_URI := "git://github.com/wistron-corporation/phosphor-nvme"
```

Comment:
This is worrisome. Have  you reached out to the phopsphor-nvme maintainers on your issues?